databricks/koalas/indexes.py

```diff
@@ -1220,14 +1231,14 @@ def droplevel(self, level):
             )
 
         int_level = [n if isinstance(n, int) else names.index(n) for n in level]
-        index_map = list(self._internal.index_map.items())
-        index_map = OrderedDict(index_map[c] for c in range(0, nlevels) if c not in int_level)
+        index_map_list = list(self._internal.index_map.items())
```

Comment:
nit: really nothing, but how about `index_map_items` ?
Maybe `index_map_list` somewhat sounds like there are several `index_map`s in the list ?

Reply:
Good idea! Renamed.